### PR TITLE
Senate yaml updates: Sen Warren, Wyden and Burr

### PR DESCRIPTION
**Pull request by OdeclasV**
Updating these US Sen yamls with new selectors, links and topics.

### CODE REVIEW

File: members/W000817_BACKUP_4580.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
+bioguide: W000817
```
Comment (annanhan) [inline members/W000817_BACKUP_4580.yaml]: do you still need this file?

File: members/W000817_BASE_4580.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
+bioguide: W000817
```
Comment (annanhan) [inline members/W000817_BASE_4580.yaml]: you should delete these files from the merge

**Comment by annanhan** [comment]
@OdeclasV you should delete the files that were generate from the merge. you only want to keep the real .yaml files

**Comment by OdeclasV** [comment]
@annanhan good catch, didn't notice those files. Done

File: members/B001135.yaml [hunk 3]
```diff
@@ -127,8 +126,8 @@ contact_form:
             Army - Private: Army - Private
             Coast Guard - Admiral: Coast Guard - Admiral
             Coast Guard - Vice Admiral: Coast Guard - Vice Admiral
-            "Coast Guard - Rear Admiral, Upper Half": "Coast Guard - Rear Admiral, Upper Half"
-            "Coast Guard - Rear Admiral, Lower Half": "Coast Guard - Rear Admiral, Lower Half"
+            Coast Guard - Rear Admiral, Upper Half": Coast Guard - Rear Admiral, Upper Half
```
Comment (tomdionesotes) [inline members/B001135.yaml]: stray quotes in lines 129 and 130

Comment (OdeclasV) [reply members/B001135.yaml]: good catch. Fixing this now

File: members/W000779.yaml [hunk 4]
```diff
@@ -3,131 +3,229 @@ contact_form:
   method: POST
   action: ""
   steps:
-    - visit: "http://www.wyden.senate.gov/contact"
+    - visit: "https://www.wyden.senate.gov/contact/email-ron"
+    - click_on:
+      - selector: div .btn-block h1
```
Comment (tomdionesotes) [inline members/W000779.yaml]: This selector is not unique to the element we want: https://gyazo.com/d61802f4baa3dbe30f413e57ca79b34c .

File: members/W000779.yaml [hunk 5]
```diff
@@ -3,131 +3,229 @@ contact_form:
...
+          "Animal Welfare": "Animal Welfare"
+          "Arts &amp; Humanities": "Arts &amp; Humanities"
+          "Banking Issues": "Banking Issues"
+          "Budget (Federal)": "Budget (Federal)"
+          "Budget (Defense)": "Budget (Defense)"
+          "Civil Rights": "Civil Rights"
+          "Climate Change": "Climate Change"
+          "Consumer Financial Protections": "Consumer Financial Protections"
+          "County Payments": "County Payments"
+          "Crime &amp; Drugs": "Crime &amp; Drugs"
```
Comment (tomdionesotes) [inline members/W000779.yaml]: encoding

File: members/W000779.yaml [hunk 6]
```diff
@@ -3,131 +3,229 @@ contact_form:
   method: POST
   action: ""
   steps:
-    - visit: "http://www.wyden.senate.gov/contact"
+    - visit: "https://www.wyden.senate.gov/contact/email-ron"
+    - click_on:
+      - selector: div .opinion
```
Comment (tomdionesotes) [inline members/W000779.yaml]: should be `div.opinion` - no space

**Comment by OdeclasV** [comment]
@tomdionesotes just updated the selector (although maybe either one would have worked). Let me know if you find any more errors, thanks!

**Comment by OdeclasV** [comment]
@tomdionesotes let me know if these are good to merge

**Pull request merged**